controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -944,7 +952,8 @@ func (r *BareMetalHostReconciler) actionPreparing(prov provisioner.Provisioner,
 	// Use settings in hostFirmwareSettings if available
 	hfs, err := r.getHostFirmwareSettings(info)
 	if err != nil {
-		info.log.Info("hostFirmwareSettings not available for cleaning")
+		// wait until hostFirmwareSettings are ready
+		return actionDelayed{}
```

Comment:
Just realized that this `actionDelayed` was primarily meant to be used in the capacity check, so I'd say that an `actionContinue{subResourceNotAvailable}` could be a better match for this case (ie, with a const `subResourceNotAvailable = time.Second * 60`)

Reply:
Good point, changed.